tools/cockpit.spec

```diff
@@ -277,6 +277,8 @@ for pkg in apps packagekit pcp playground storaged; do
 done
 # files from -tests
 rm -f %{buildroot}/%{pamdir}/mock-pam-conv-mod.so
+rm -f ${buildroot}/%{_unitdir}/cockpit-session.socket
+rm -f ${buildroot}/%{_unitdir}/cockpit-session@.service
```

Comment:
Wrong currency, it's not dollars, but you only get a percentage :grin: 

This fixes the [unpackaged files](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16819-20230104-101153-3466f0a9-rhel-8-8/log.html) failure.

The [opposite](https://cockpit-logs.us-east-1.linodeobjects.com/pull-16819-20230104-101153-3466f0a9-rhel-8-8-distropkg/log.html) happens because "when not building basic packages, remove their files" removes the entirety of %{buildroot}/%{_prefix}/%{__lib}/systemd. I'll get that fixed as well.

Reply:
Fixed both.